lib/pages/reports/bite/pages/moment_question_page.dart

```diff
@@ -0,0 +1,164 @@
+import 'package:flutter/material.dart';
+import 'package:mosquito_alert/mosquito_alert.dart';
+import 'package:mosquito_alert_app/utils/MyLocalizations.dart';
+import 'package:mosquito_alert_app/utils/style.dart';
+
+class EventMomentPage extends StatefulWidget {
+  final void Function(BiteRequestEventMomentEnum) onNext;
+  final VoidCallback onPrevious;
+  const EventMomentPage(
+      {super.key, required this.onNext, required this.onPrevious});
+
+  @override
+  State<EventMomentPage> createState() => _EventMomentPageState();
+}
+
+class _EventMomentPageState extends State<EventMomentPage> {
+  String? _mainSelection; // "just_now" or "last_24h"
```

Comment:
have you considered using an enum instead of a string literal?


```suggestion
  MainSelectionType _mainSelection;// "just_now" or "last_24h"
```

and a declared enum like:
```
enum MainSelectionType { justNow, last24h, none }
```

Or maybe there's an enum already declared in the sdk

Reply:
Cool! I've just commited: https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/pull/627/commits/4c5ca39b3230fb2b000edcc678441ce9dcaaebab